sound/soc/sof/sof-audio.h

```diff
@@ -392,6 +392,13 @@ struct snd_sof_widget {
 	int num_sink_pins;
 	int num_source_pins;
 
+	/* The sink/source pin binding array, it takes the form of
+	 * [widget_name_connected_to_pin0, widget_name_connected_to_pin1, ...],
```

Comment:
The comment here is not valid now, right?

Reply:
I think it is valid, the pin_binding is still an array, it is in the form of char **